blocks/breadcrumb/breadcrumb.css

```
@@ -0,0 +1,54 @@
+/* Breadcrumb CSS */
+
+.section.breadcrumb-container > div.breadcrumb-wrapper {
+  padding: unset;
+  width: 100%;
+  background-color: #F3F1ED;
```

```suggestion
  background-color: var(--neutral-40);
```

Let's add this neutral 40% hex color variable to our `style.css` - `--neutral-40: F3F1ED;`